doc/usecases/done/search_plants.md

```diff
@@ -41,3 +41,7 @@
 - **Non-functional Constraints:**
   - Performance
   - If there is a possible match in the database, it should be included (regardless of language settings etc.)
+
+# Followup
+Some parts of this use cases have not been implemented in preperation for a future university course.
+Followup issues have been created in GitHub, outlining what work still needs to be done.
```

Comment:
Please link these issues here and please also link the protocol. See doc/usecases/README.md for which links are expected.

```suggestion

```